lib/safe_pusher.rb

```diff
@@ -1,16 +1,19 @@
 require 'safe_pusher/configuration'
 require 'thor'
 require 'colorize'
-require 'i18n'
 require 'English'
+require 'i18n'
 require 'yaml'
+require 'fileutils'
 require 'safe_pusher/cli'
 require 'safe_pusher/version'
 require 'safe_pusher/client/git'
 require 'safe_pusher/client/rspec'
 require 'safe_pusher/client/pronto'
 require 'safe_pusher/client/github'
 
+I18n.load_path << Dir[File.expand_path('config/locales') + '/*.yml']
```

Comment:
You should be able to expand paths with globs:

```suggestion
I18n.load_path << Dir[File.expand_path('config/locales/*.yml')]
```